### ISSUE TITLE
Keyboard navigation doesn't work in style list in Z7 Document Preferences window

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#3214]

**Comment by adomasven** [comment]
Works for me. It's even focused upon window open. Steps to reproduce?

**Comment by dstillman** [comment]
Sorry, to be clear, I mean FAYT, not arrow keys.
